templates/tab/ts/default/.env.teamsfx.local

```diff
@@ -0,0 +1,13 @@
+# TeamsFx will overwrite the following variable values when running debug
+SSL_CRT_FILE=
+SSL_KEY_FILE=
+PORT=
+BROWSER=none
+HTTPS=true
```

Comment:
@swatDong Shall we keep the values (`none`, `true`) like the original `.env` file does, or just remove the values as well?

Reply:
I'd like to keep these, since teams tab app requires https.